src/cli.js

```diff
@@ -17,7 +17,18 @@ sourceMaps.install({
       return;
     }
     const realPath = filePath.slice(prefix.length);
-    return fs.readFileSync(realPath, 'utf-8');
```

Comment:
We could do an `if (fs.existsSync(realPath)) { ... }` instead and avoid the try-catch (right?). The try/catch is fine too though.

Reply:
The Node docs recommend handling errors instead of checking for existence (at least for the async versions), so I decided to go that route here. Probably doesn't make much difference either way, but I suspect the try/catch might perform a bit better for the happy path?

https://nodejs.org/api/fs.html#fs_fs_exists_path_callback